src/LighthouseServiceProvider.php

```diff
@@ -71,15 +68,6 @@ public function boot(ValidationFactory $validationFactory, ConfigRepository $con
         ], 'schema');
 
         $this->loadRoutesFrom(__DIR__.'/Support/Http/routes.php');
-
-        $validationFactory->resolver(
-            function ($translator, array $data, array $rules, array $messages, array $customAttributes): Validator {
-                // This determines whether we are resolving a GraphQL field
-                return Arr::has($customAttributes, ['root', 'context', 'resolveInfo'])
-                    ? new GraphQLValidator($translator, $data, $rules, $messages, $customAttributes)
-                    : new Validator($translator, $data, $rules, $messages, $customAttributes);
-            }
-        );
```

Comment:
Nice that this got removed!

Reply:
Yeah. This PR really cleans up a lot of cruft from previous attempts and variants to do validation. Felt really good to delete that.